lib/src/cancelable_operation.dart

```diff
@@ -202,7 +209,9 @@ class CancelableCompleter<T> {
     });
   }
 
-  /// Completes [operation] to [error].
+  /// Completes [operation] with [error].
```

Comment:
`... with [error] and [stackTrakce].`

There is no mention of the `stackTrace` parameter at all.